cloud.ganttproject.colloboque/src/main/kotlin/cloud/ganttproject/colloboque/Postgres.kt

```diff
@@ -57,11 +54,11 @@ class PostgresDataSourceFactory(
         stmt.execute()
       }
     }
-    return HikariDataSource(HikariConfig().apply {
-      username = pgSuperUser
-      password = pgSuperAuth
-      jdbcUrl = "jdbc:postgresql://${pgHost}:${pgPort}/dev_all_projects"
-      this.schema = schema
-    }).also { cachedDataSources[projectRefid] = it }
   }
+
+  fun createConnection(projectRefid: String): Connection =
```

Comment:
super data source was intended for metadata manipulations by a super-user, such as storing instance-level mappings of projects to databases, or cloning a template database/template schema. I suggest using a separate "non-super" datasource for the regular connections